docs/rfc/002_standard_memo_format.md

```diff
@@ -0,0 +1,131 @@
+<h1 style="text-align: center;">A proposed standard memo format for non-EVM chains</h1>
+<br>
+
+## The goal
+
+This doc is aimed to introduce a standard format of ZetaChain inbound `memo` across non-EVM chains. The `memo` in an inbound transaction is to carry both instructions and data, by which the cross-chain transaction to be processed as expected in the `zetacore`. A standard memo format helps to ensure consistent developer experience across EVM and non-EVM chains.
+<br><br>
+
+
+## The EVM gateway inbound events
+
+We want to mimic below `Deposit` and `Called` events in the EVM `gateway` contract for non-EVM chains. Typically, we at least want the following 3 pieces at the time of writing.
+
+- The `receiver` address in ZEVM.
+- The `payload` of the call.
+- The `RevertOptions`.
+
+```
+event Deposited(
+    address indexed sender,
+    address indexed receiver,
+    uint256 amount,
+    address asset,
+    bytes payload,
+    RevertOptions revertOptions
+);
+
+event Called(
+	   address indexed sender,
+	   address indexed receiver,
+	   bytes payload,
+	   RevertOptions revertOptions
+);
```

Comment:
NIT : seems like events need to be aligned

Reply:
https://github.com/zeta-chain/node/pull/2963/commits/c292962e5709dfc4ed0b94e5c6f609c680fff2e7